setup.py

```diff
@@ -174,8 +183,6 @@ def initialize_options(self):
     def run(self):
         global BUILD_DIR, BUILT_EXTENSIONS, EIGEN3_INCLUDE_DIR
         BUILD_DIR = os.path.abspath(self.build_dir)
-        if EIGEN3_INCLUDE_DIR is None:
```

Comment:
This breaks the `pip` install because the `hg` below clones it into the current directory, which is the updated `BUILD_DIR` and not the original one (e.g., `.../py-3.5-x64/build` and not `../build`).